worker/src/handlers/aggregationProcessor/aggregationProcessor.js

```diff
@@ -275,7 +310,8 @@ const aggregationProcessor = async (
   {
     aggregationProcessorId,
     publishQueue = publish,
-    now // For testing
+    now, // For testing
+    actualNow // for testing
```

Comment:
Can we give these two variables clearer names? Perhaps rename `now` to `endDate` and `actualNow` to `now` or something similar?

Reply:
Yes, I didn't really come up with better names. Now is where the window is calculated from, so the case where this wouldn't be the same as actualNow is if benchmarking is enabled. Then 'now' would be actualNow - windowSize.

But I want to give it some obscure long name, as now is the most import variable in the algorithm, where as actualNow is just to catch the edge case of future insertions.